### PR TITLE
Update MainAPI.kt

### PR DESCRIPTION
**Pull request by redowan99**
Adding getYearFromString()

### CODE REVIEW

File: library/src/commonMain/kotlin/com/lagradost/cloudstream3/MainAPI.kt [hunk 1]
```diff
@@ -1217,6 +1217,28 @@ interface LoadResponse {
...
+ * "This is 2023 movie" -> 2023
+ *
+ * "This is 1999-2010 TvSeries" -> 1999
+ *
+ * @param check The input string.
+ * @return The year as an integer, or `null` if no match is found.
+ */
+fun getYearFromString(check: String?): Int? {
+    return check?.let {
+        "(?<=\\()\\d{4}(?=\\))".toRegex().find(it)?.value?.toIntOrNull()
```
Comment (fire-light42) [inline library/src/commonMain/kotlin/com/lagradost/cloudstream3/MainAPI.kt:1238]: Regex construction makes this function take ~10x longer than it should. As such, I would like if you moved both of the .toRegex() into private variables in the global scope

**Review by fire-light42 — CHANGES_REQUESTED** [review]

**Comment by fire-light42** [comment]
Why was this closed?

**Comment by C10udburst** [comment]
silly

**Comment by redowan99** [comment]
I didn't know deleting the fork, would close the commit. I think I need to learn to work with git first🤦‍♂️.

**Pull request closed without merging**